docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -161,6 +161,16 @@ values. Typical usage does not require you to configure them.
   - Maximum number of metastore data objects per transaction in the Hive
     metastore cache.
   - `1000`
+* - `delta.metastore.store-table-metadata-enabled`
+  - Store table comments and colum definitions to the metastore. The write
+    permission is required to update the metastore.
+  - `true`
+* - `delta.metastore.store-table-metadata-threads`
+  - Number of threads used for internal task store table metadata in metastore.
```

Comment:
@mosabua can you help with wording here

Reply:
I would wait for @mosabua feedback here and `delta.metastore.store-table-metadata-interval` description.